common/app/views/support/EmailHelpers.scala

```diff
@@ -50,7 +50,7 @@ object EmailHelpers {
   }
 
   def icon(name: String) = Html {
-    s"""<img src="${Static(s"images/email/icons/$name.png")}" class="float-left icon icon-$name">"""
+    s"""<img src="${Static(s"images/email/icons/$name.png")}" class="icon icon-$name">"""
```

Comment:
wow crazy, it seems like that would be _harder_ to implement than support! fair enough!